src/Tests/SNIPPETS.cs

```diff
@@ -94,6 +94,31 @@ @ EntryPoint (  )
     }
 ";
 
+        public static string OpenNamespaces1 =
+@"
+    open Microsoft.Quantum.Intrinsic;
```

Comment:
Should there be a test to make sure that `open X and Y` is either supported, or fails with an actionable error message?

Reply:
Thanks for pointing this out! I hadn't considered the case of `open X as Y` directives. Yes, looks like I need to add support to avoid breaking that, ignore one more warning, and then I'll add tests for this case.